force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -70,6 +76,11 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Decimal division(Integer a, Integer b) {
         return null; // Replace null with the variable you used to store the result
+        if (b == 0) {
+            return 0;
+        } else {
+             return (Decimal) a/b;
```

Comment:
This is an example of implicit conversion. These two integers will get automatically converted to the Decimal return type without you having to explicitly cast.